tools/lib-alert-tree/lib_alert_tree/kubernetes.py

```diff
@@ -3,14 +3,24 @@
 from . import models
 
 
+def container_alerts(name, pod=".*", severity="warning", namespace="default"):
```

Comment:
I suggest you put a default on `name` as well so that (I agree) the call in `pod_alerts` will be more readable

```suggestion
def container_alerts(name=".*", pod=".*", severity="warning", namespace="default"):
```